### ISSUE TITLE
Accessibility: Checkboxes need 'aria-labelled by' tag to indicate what content they are associated with

### ISSUE THREAD

**Issue by astridu** [sul-dlss/happy-heron#1750]
Issue: If the input field has a visible description indicating the purpose of the field, this description should be explicitly associated to the input field either as a HTML label or using the WAI-ARIA attribute 'aria-labelledby'.

**Comment by jcoyne** [comment]
@astridu can you identify a particular checkbox that does not have a label tag? I believe all the checkboxes have labels.

**Comment by astridu** [comment]
The SiteImprove accessibility checker is asking for the 'aria-labelledby' tag, but if our label tags do the same thing, then I can close this.

**Comment by jcoyne** [comment]
Yes, label tags are the classical way of handling this. Of course there may be some that don't have a label, and we'd want to find those instances.

**Comment by astridu** [comment]
Let me look.

**Comment by astridu** [comment]
Ok, it's not finding any that don't have a label.
